src/Storefront/Resources/views/storefront/component/line-item/type/product.html.twig

```diff
@@ -26,12 +26,17 @@
             {% block component_line_item_type_product_col_info %}
                 <div class="line-item-info">
                     <div class="row line-item-row">
+                        {% set lineItemLink = seoUrl('frontend.detail.page', { 'productId': lineItem.referencedId }) %}
```

Comment:
Do we need to set those variables here when the defaults/fallbacks are handled in the image and label template? I would find it easier to understand when the variables are only explicitly passed in the includes when needed. But I am not sure if wee need it here?

Reply:
My first implementation is found here: https://github.com/shopware/platform/commit/b723078acdf48612d5f8cf6a9a0331d53237fe1c

I just added the `productLink` back as a response to this comment: https://github.com/shopware/platform/pull/2974#discussion_r1103808095 

I think this is related to the other discussion.